### PR TITLE
GCS:  Welcome Page update HITL Button to point to Advanced view

### PR DESCRIPTION
**Pull request by VArcht**
HITL button on the welcome page did not do anything when clicked now it
will navigate to advanced view

[Review on Reviewable](https://reviewable.io/reviews/d-ronin/dronin/199)

### CODE REVIEW

File: ground/gcs/src/plugins/welcome/qml/main.qml [hunk 1]
```diff
@@ -90,8 +90,8 @@ Rectangle {
 
                 WelcomePageButton {
                     baseIconName: "hitl"
```
Comment (tracernz) [inline ground/gcs/src/plugins/welcome/qml/main.qml]: Should really rename this too.

Comment (VArcht) [inline ground/gcs/src/plugins/welcome/qml/main.qml]: Breaks it, if it is important I'll have to dig into it, wasn't immediately obvious how it works.

Comment (tracernz) [inline ground/gcs/src/plugins/welcome/qml/main.qml]: I imagine it points to the image filename?

Comment (VArcht) [inline ground/gcs/src/plugins/welcome/qml/main.qml]: Yeah probably, I did consider that before I got lazy. I'll give it a look in the morning, thx.

Comment (tracernz) [inline ground/gcs/src/plugins/welcome/qml/main.qml]: Yeah, it does: https://github.com/d-ronin/dRonin/blob/next/ground/gcs/src/plugins/welcome/qml/WelcomePageButton.qml#L16

**Comment by VArcht** [comment]
I addressed the icon name.

**Pull request merged**